start.go

```diff
@@ -202,8 +202,7 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 				"--state-dir", root,
 				"--listen", filepath.Join(namespace, "namespaced.sock"),
 			)
-			cmd = exec.Command("runv", args...)
-			cmd.Path = path
+			cmd = exec.Command(path, args...)
```

Comment:
@laijs  Actually from the source code of golang os/exec package, we could find that  if cmd.Env is nil, system will transfer os.Environ to child by default.
[https://golang.org/src/os/exec/exec.go?s=4289:4334#L119](url)
[https://golang.org/src/os/exec/exec.go?s=4289:4334#L119](url)